test/todolist_web/controllers/page_controller_test.exs

```diff
@@ -1,8 +1,25 @@
 defmodule TodolistWeb.PageControllerTest do
   use TodolistWeb.ConnCase
 
-  test "GET /", %{conn: conn} do
-    conn = get(conn, "/")
+  setup %{conn: conn} = config do
+    if config[:logged_in] == true do
```

Comment:
I think you don't need this whole if-else block.
Since you create 2 types of connections (one auth and one not), you can easily only return
```elixir
{:ok, conn: conn, authed_conn: authed_conn}
```

Reply:
Took me a while to see your point. I was too bent on using the `@tag` feature.

Reply:
Yep, sometimes the less exciting way ends up being cleaner 😅